testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/web/security/authentication/BasicAuthenticationMechanismPicketboxRemovedTestCase.java

```diff
@@ -0,0 +1,93 @@
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.jboss.as.test.integration.web.security.authentication;
+
+import org.apache.http.HttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.jboss.as.arquillian.api.ServerSetup;
+import org.jboss.as.test.integration.security.common.Utils;
+import org.jboss.as.test.integration.web.security.authentication.deployment.SecuredEJB;
+import org.jboss.as.test.integration.web.security.authentication.deployment.SecuredEJBServlet;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.net.URL;
+
+/**
+ * Automated test for [ WFLY-10282 ]
+ * Test Enables elytron and removes security subsystem. After secured EJB is deployed and accessed using basic authorization
+ * test checks if no NullPointerException is thrown.
+ *
+ * Test is also related to the issue [ WFLY-10292 ]: after switching to elytron and removing picketbox subsystem DefaultJMSConnectionFactory is not found during server startup.
+ *
+ * @author Daniel Cihak
+ */
+@RunWith(Arquillian.class)
+@ServerSetup(BasicAuthMechanismServerSetupTask.class)
+public class BasicAuthenticationMechanismPicketboxRemovedTestCase {
+
+    private static final String EJB_SECURITY = "ejb-security";
+
+    @Deployment(name = EJB_SECURITY)
+    public static WebArchive appDeployment1() {
+        WebArchive war = ShrinkWrap.create(WebArchive.class, EJB_SECURITY + ".war");
+        war.addClasses(BasicAuthenticationMechanismPicketboxRemovedTestCase.class, SecuredEJB.class, SecuredEJBServlet.class);
+        war.setWebXML(Utils.getResource("org/jboss/as/test/integration/web/security/authentication/deployment/web.xml"));
+        war.addAsWebInfResource(Utils.getResource("org/jboss/as/test/integration/web/security/authentication/deployment/jboss-web.xml"), "jboss-web.xml");
```

Comment:
This is unnecessary, use .addAsWebInfResource(BasicAuthenticationMechanismPicketboxRemovedTestCase.class.getPackage(), "jboss-web.xml", "jboss-web.xml"), same applies for web.xml.

Arquillian is designed to load resources from the class path, there is no need to use utility methods for this.